sourcemash/frontend/static/js/collections/items.js

```diff
@@ -1,12 +1,17 @@
 Sourcemash.Collections.Items = Backbone.Collection.extend({
     model: Sourcemash.Models.Item,
     url: function() {
-        return '/api/feeds/' + this.feed.get('id') + '/items'
+        if (this.feed) {
+            return '/api/feeds/' + this.feed.get('id') + '/items'
+        } else if (this.category) {
+            return '/api/categories/' + this.category.get('category') + '/items'
+        }
```

Comment:
Is it every possible for both this.feed and this.category to be set simultaneously? How does it know which one to look at?